tools/msvs/install_tools/install_tools.bat

```diff
@@ -0,0 +1,25 @@
+@echo off
+
+echo ====================================================
+echo Tools for Node.js Native Modules Installation Script
+echo ====================================================
+echo.
+echo This Boxstarter script will install Python and the Visual Studio Build Tools,
+echo necessary to compile Node.js native modules. All necessary Windows updates
+echo will also be installed.
+echo.
+echo This will require about 3 Gb of free disk space, plus any space necessary to
+echo install Windows updates.
+echo.
+echo This will take a while to run. Your computer may reboot during the
+echo installation, and will resume automatically.
+echo.
+echo Please close all open programs for the duration of the installation.
+echo.
+echo You can close this window to stop now. This script can be invoked from the
+echo Start menu. Detailed instructions to install these tools manually are
+echo available at https://github.com/nodejs/node-gyp#on-windows
+echo.
+pause
+
+"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('http://boxstarter.org/bootstrapper.ps1')); get-boxstarter -Force; Install-BoxstarterPackage -PackageName '%~dp0\install_tools.txt'"
```

Comment:
The script location is `http://boxstarter.org/bootstrapper.ps1` could we use the `https` flavor?
Is there a GitHub or versioned flavor of the script so we can see changes _(since this is an arbitrary third-party url it could load anything)_.

Reply:
Yes please on https!

Reply:
https://boxstarter.org/bootstrapper.ps1

Reply:
That was probably from an artifact when there was not TLS on the site (we added that to boxstarter in May when we took over).

Reply:
From https://github.com/nodejs/node/pull/22645#issuecomment-417896340:

@joaocgreis I think the instructions now look updated... whether they were when you first saw this or not is out of my knowledge.

Reply:
@ferventcoder I think https://github.com/nodejs/node/pull/24237 is the most recent PR that deals with this. I'll adopt your suggestion.

Reply:
NM it's already there
https://github.com/nodejs/node/blob/5c201b6d477525a080b11d1bd49a7c94fe81f8ac/tools/msvs/install_tools/install_tools.bat#L76